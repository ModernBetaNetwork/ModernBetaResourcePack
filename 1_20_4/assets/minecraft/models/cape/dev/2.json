{
	"parent": "minecraft:cape/cape_2",
	"textures": {
		"0": "item/capes/dev",
		"particle": "item/capes/dev"
	}
}