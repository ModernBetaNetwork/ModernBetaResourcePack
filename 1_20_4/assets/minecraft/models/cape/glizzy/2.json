{
	"parent": "minecraft:cape/cape_2",
	"textures": {
		"0": "item/capes/glizzy",
		"particle": "item/capes/glizzy"
	}
}