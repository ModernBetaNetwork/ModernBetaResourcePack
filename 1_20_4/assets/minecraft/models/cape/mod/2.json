{
	"parent": "minecraft:cape/cape_2",
	"textures": {
		"0": "item/capes/mod",
		"particle": "item/capes/mod"
	}
}