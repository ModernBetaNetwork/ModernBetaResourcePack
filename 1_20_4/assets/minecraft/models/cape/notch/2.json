{
	"parent": "minecraft:cape/cape_2",
	"textures": {
		"0": "item/capes/notch",
		"particle": "item/capes/notch"
	}
}