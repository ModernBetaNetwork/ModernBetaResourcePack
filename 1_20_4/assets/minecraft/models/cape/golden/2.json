{
	"parent": "minecraft:cape/cape_2",
	"textures": {
		"0": "item/capes/golden",
		"particle": "item/capes/golden"
	}
}